dist/fabric.js

```diff
@@ -19501,6 +19584,8 @@ fabric.util.object.extend(fabric.Object.prototype, /** @lends fabric.Object.prot
     }
   };
 
+  fabric.Path.NUM_PATH_DIGITS = 0;
```

Comment:
I know you say its a loseless compression, but would it still be worth it to add some option to not use it?

Reply:
The path simplification is not compression, it's just an optimization. It removes redundant points when there is a straight line composed of more than two points.

Reply:
Well this is not lossless... as it trims precision. My understanding is 1 = 1 pixel and so trimming to int's (aka 0 precision) sounds good. 

I don't think we need to control this dynamically, but that said, can't the app overwrite this value on startup if it wanted to?

Reply:
Wait...there are two things here. 
1) The precision, which is configurable and IS lossy 
2) The 'simplify paths' which is NOT configurable, but is lossless (in theory)

Reply:
Yup!

I do think we're ok here:
1) Can be set by the app 
2) Lossless. This is similar to the decimate functionality. This is a property on the PencilBrush. Not sure where Max your setting simplifyPath, but as its a member var it seems like something the app could control since it creates the brush right? That said I think it'd be fine as a static config in the lib, but do think its settable but will let Max confirm.

Reply:
@msucharski I see you removed the if in front of simplifyPath (and yeah, i see it was checking the method presence). Only remaining issue on this PR is that if Olivier feels strongly we should be able to turn it off, then it might be worth just adding a member var flag (a-la decimate)?

Reply:
@brassaro - Do you feel like this needs a flag? Happy to add one if so, but I don't think it needs to exist!

Reply:
I don't feel strongly about it. You win!